src/cljs/ataru/virkailija/editor/component.cljs

```diff
@@ -182,6 +182,26 @@
               :fade-in  "animated fadeInUp"
               nil)))
 
+(defn- remove-component-button [component-wrapped? path]
+  (case @(subscribe [:editor/remove-component-button-state path])
+    :active
+    [:button.editor-form__remove-component-button
+     {:on-click #(dispatch [:editor/start-remove-component path])}
+     "Poista"]
+    :confirm
+    [:button.editor-form__remove-component-button--confirm.editor-form__remove-component-button
+     {:on-click (fn [event]
+                  (let [target (if component-wrapped?
```

Comment:
The `.-parentNode ..` thing seems a bit fragile. Could e.g. `getAncestor` from https://google.github.io/closure-library/api/goog.dom.html be used?